android/java/org/chromium/chrome/browser/crypto_wallet/activities/BuySendSwapActivity.java

```diff
@@ -985,28 +956,21 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
                                 setSendToFromValueValidationResult(errorMessage, false, true);
                             });
                 }
-            } else if (mActivityType == ActivityType.BUY) {
-                if (mSelectedNetwork.chainId.equals(BraveWalletConstants.MAINNET_CHAIN_ID)) {
-                    assert mBlockchainRegistry != null;
-                    String asset = mFromAssetText.getText().toString();
-                    mBlockchainRegistry.getBuyUrl(OnRampProvider.WYRE,
-                            BraveWalletConstants.MAINNET_CHAIN_ID, from, asset, value,
-                            (url, error) -> {
-                                if (error != null && !error.isEmpty()) {
-                                    Log.e(TAG, "Could not get buy URL: " + error);
-                                    return;
-                                }
+            } else if (mActivityType == ActivityType.BUY
+                    && mSelectedNetwork.chainId.equals(BraveWalletConstants.MAINNET_CHAIN_ID)) {
+                assert mBlockchainRegistry != null;
+                String asset = mFromAssetText.getText().toString();
+                mBlockchainRegistry.getBuyUrl(OnRampProvider.WYRE,
+                        BraveWalletConstants.MAINNET_CHAIN_ID, from, asset, value,
+                        (url, error) -> {
+                            if (error != null && !error.isEmpty()) {
+                                Log.e(TAG, "Could not get buy URL: " + error);
```

Comment:
nit: @SergeyZhukovsky one mentioned that we should remove these old logs so can remove.

Reply:
Yeah, we better limit any information we expose to logcat to avoid possible leaks of sensitive info. I would recommend or detect if it's a debug mode and print log to help troubleshooting by devs or do assert on such places so they trigger and we can go from there.

Reply:
Added a debug build check.